C/include/c4Index.h

```diff
@@ -11,6 +11,8 @@
 //
 
 #pragma once
+#include "c4Base.h"
+#include "c4Compat.h"
```

Comment:
That's not necessary. Those headers are transitively included by c4IndexTypes.h, and none of the other headers (to my knowledge) redundantly include them.

Reply:
removed